app/views/shelters/new.html.erb

```diff
@@ -1,15 +1,15 @@
-<form action="/shelters" method="post" class="new-shelter">
-    <input type="hidden" name="authenticity_token" value="<%=form_authenticity_token %>">
-    <p>Enter information for a new shelter:</p>
-    <label>Name</label>
-    <input type="text" name='shelter[name]' required /><br/>
-    <label>Address</label>
-    <input type="text" name='shelter[address]' required/><br/>
-    <label>City</label>
-    <input type="text" name='shelter[city]' required/><br/>
-    <label>State</label>
-    <input type="text" name='shelter[state]' required/><br/>
-    <label>Zip</label>
-    <input type="text" name='shelter[zip]' required/><br/>
-    <input type="submit" value="submit"/>
-</form>
\ No newline at end of file
+<h2>New Shelter</h2>
+
+<%= form_tag "/shelters", method: :post do %>
+    <%= label_tag :name %>
+    <%= text_field_tag :name %>
+    <%= label_tag :address %>
+    <%= text_field_tag :address %>
+    <%= label_tag :city %>
+    <%= text_field_tag :city %>
+    <%= label_tag :state %>
+    <%= text_field_tag :state %>
+    <%= label_tag :zip %>
+    <%= text_field_tag :zip %>
+    <%= submit_tag "Submit" %>
+<% end %>
```

Comment:
The old form was replaced using the form_tag in rails rather than using regular html.